test/testsuites/examples/create-eks.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-eks-task
+  namespace: tekton
+spec:
+  description: |
+    Create an EKS cluster and MNG.
+  params:
+  - name: cluster-name
```

Comment:
So you want the name to be `create-eks-{random_gen}`? Since the random-gen naming that's done by Tekton is on runtime this is hard to figure out. We can randomly parameterize this in a future PR.